adapters/consors-bank-adapter/src/main/java/de/adorsys/xs2a/adapter/consors/ConsorsPaymentInitiationService.java

```diff
@@ -0,0 +1,50 @@
+package de.adorsys.xs2a.adapter.consors;
+
+import de.adorsys.xs2a.adapter.api.RequestHeaders;
+import de.adorsys.xs2a.adapter.api.RequestParams;
+import de.adorsys.xs2a.adapter.api.Response;
+import de.adorsys.xs2a.adapter.api.http.HttpClient;
+import de.adorsys.xs2a.adapter.api.link.LinksRewriter;
+import de.adorsys.xs2a.adapter.api.model.Aspsp;
+import de.adorsys.xs2a.adapter.api.model.PaymentInitationRequestResponse201;
+import de.adorsys.xs2a.adapter.api.model.PaymentProduct;
+import de.adorsys.xs2a.adapter.api.model.PaymentService;
+import de.adorsys.xs2a.adapter.impl.BasePaymentInitiationService;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Map;
+import java.util.function.Function;
+
+import static de.adorsys.xs2a.adapter.api.RequestHeaders.PSU_ID;
+
+public class ConsorsPaymentInitiationService extends BasePaymentInitiationService {
+
+    public ConsorsPaymentInitiationService(Aspsp aspsp,
+                                           HttpClient httpClient,
+                                           LinksRewriter linksRewriter) {
+        super(aspsp, httpClient, linksRewriter);
+    }
+
+    @Override
+    protected <T> Response<PaymentInitationRequestResponse201> initiatePayment(PaymentService paymentService,
```

Comment:
Typically adapters override only public methods, those that are protected are intended for delegation when some custom mapping logic has to be applied.

Reply:
got it, thx